packages/bpk-component-autosuggest/src/BpkAutosuggest-test.js

```diff
@@ -26,7 +26,12 @@ const onSuggestionsFetchRequested = () => null;
 const onSuggestionsClearRequested = () => null;
 const getSuggestionValue = suggestion => suggestion;
 const renderSuggestion = suggestion => <span>{suggestion}</span>;
-const inputProps = { value: 'Edinburgh', onChange: () => null };
+const inputProps = {
+  id: 'origin',
+  name: 'Origin',
```

Comment:
I had to add `id` and `origin` to the input props because they are required in `BpkInput`. Previously these prop type warnings were not being raised _because_ we were not implementing `renderInputComponent` correctly.

I think the term for what was happening is _squashing_ 🤔